src/displayapp/screens/Dice.h

```diff
@@ -3,10 +3,10 @@
 #include "components/motion/MotionController.h"
 #include "displayapp/screens/Screen.h"
 #include "displayapp/widgets/Counter.h"
-#include "lvgl/lvgl.h"
 #include "systemtask/SystemTask.h"
 
 #include <array>
+#include <lvgl/lvgl.h>
```

Comment:
I'm probably starting to go on your nerves. Sorry about that :sweat_smile: 

first system header, then std-lib-header

```suggestion
#include <lvgl/lvgl.h>

#include <array>
```

Reply:
Not in the least :smile: I appreciate your time & patience going thru the code & making it better!